### PR TITLE
ARROW-4184: [Ruby] Add Arrow::RecordBatch#to_table

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Review by shiro615 — DISMISSED** [review]
LGTM
I'll merge this when CI is passed.

File: ruby/red-arrow/test/test-record-batch.rb [hunk 1]
```diff
@@ -54,4 +54,9 @@ class RecordBatchTest < Test::Unit::TestCase
                    records.collect {|record, i| [record.index, i]})
     end
   end
+
+  test("#to_table") do
+    assert_equal(Arrow::Table.new(@schema, [@counts]),
```
Comment (shiro615) [inline ruby/red-arrow/test/test-record-batch.rb]: It seems necessary to use 3.3.1 version of Ruby/GObjectIntrospection to pass an array of GObject.
CI is using 3.3.0.
https://travis-ci.org/apache/arrow/jobs/476565787#L6837

Comment (kou) [reply ruby/red-arrow/test/test-record-batch.rb]: Good catch!
I've released gobject-introspection gem 3.3.1.
I changed to require gobject-introspection gem 3.3.1 or later.

**Review by shiro615 — APPROVED** [review]
LGTM
I'll merge this.

**Pull request closed without merging**